mon-pix/tests/unit/components/password-reset-demand-form-test.js

```diff
@@ -17,54 +18,55 @@ describe('Unit | Component | password-reset-demand-form', () => {
 
     beforeEach(function() {
       saveStub = sinon.stub().resolves();
-      createRecordStub = sinon.stub().returns({
-        save: saveStub
+      createRecordStub = sinon.stub().resolves({
+        save: saveStub,
+        toto: () => 42
       });
 
-      component = this.owner.lookup('component:password-reset-demand-form');
-      component.set('store', Service.create({
+      component = createGlimmerComponent('component:password-reset-demand-form');
+      component.store = Service.create({
```

Comment:
Il me semble que ça doit être comme ça 
```suggestion
	  store = = Service.create({
	  	createRecord: createRecordStub
	  });
	  email = sentEmail;
      component = createGlimmerComponent('component:password-reset-demand-form', { store, url });
```

Reply:
il me semble que l'objet en 2eme argument correspond uniquement aux paramètres du composant, pas aux dépendances injectées.
En l'occurence les tests passent tout bien :-)